ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java

```diff
@@ -77,4 +80,15 @@ private static void makeAccessible(final Field field) {
 		}
 	}
 
+	public static List<Field> getDeclaredFieldsUpToParent(Class<?> currentClass, Class<?> exclusiveParent) {
```

Comment:
What is ```exclusiveParent``` ?
Is it really necessary.

And just ```getDeclaredFieldsIncludingParent``` might be  better name.

* currentClass => current
* parentClass => parent

Reply:
It is optional.
ngirnder may be not use this.

is better remove this?